tests/contract/test_get_specific_catalog.py

```diff
@@ -7,17 +7,6 @@
 from tests import responses
 
 
-@pytest.mark.contract
-@pytest.mark.docker
-def test_ramsund(docker_service: str) -> None:
```

Comment:
Dropper denne siden det er rimelig umulig å mocke dato for disse testene, den kjøres uansett som `integration`